### PR TITLE
All responses return a 200 OK status code #5

### PR DESCRIPTION
**Pull request by VictorAlagwu**
Try and see if it is functional

### CODE REVIEW

File: public/cookie.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Netscape HTTP Cookie File
```
Comment (shalvah) [inline public/cookie.txt]: This file should be added to the gitignore

**Comment by shalvah** [comment]
Nice work! I'll drop a few comments on style. While we don't have any official code style guidelines, these days I generally try to stick to [PSR-2](https://www.php-fig.org/psr/psr-2/). I might set up CI that checks for that later. For now, I'll drop some comments on specific places. Thanks!

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -49,7 +49,7 @@ public function render($request, Exception $e)
         if($e instanceof FatalThrowableError){
             $response = array("status" => "failed",
                 "message" => "authentication failed: incorrect username or password");
-            return response(json_encode($response));
+            return response()->json($response,400);
```
Comment (shalvah) [inline app/Exceptions/Handler.php:52]: Please leave a space between method parameters:

```
...json($response, 400);
```

File: app/Http/Controllers/ApiController.php [hunk 3]
```diff
@@ -18,13 +18,13 @@ public function __construct()
     public function getStudentDetails(Request $request)
     {
         if (!($request->has("username"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing username"));
+            $userResponse = array("status" => "Fail", "message" => "missing username");
+            return response()-> json($userResponse, 400);
```
Comment (shalvah) [inline app/Http/Controllers/ApiController.php]: Please remove the space before ` json(`

File: app/Http/Controllers/ApiController.php [hunk 4]
```diff
@@ -18,13 +18,13 @@ public function __construct()
     public function getStudentDetails(Request $request)
     {
         if (!($request->has("username"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing username"));
+            $userResponse = array("status" => "Fail", "message" => "missing username");
```
Comment (shalvah) [inline app/Http/Controllers/ApiController.php]: We can actually just call this `$response`. `$userResponse` is misleading. My first impression was that it contained the user's details.

File: app/Http/Controllers/ApiController.php [hunk 5]
```diff
@@ -18,13 +18,13 @@ public function __construct()
...
+            return response()-> json($userResponse, 400);
         }
 
         if (!($request->has("password"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing password"));
+            $passResponse = array("status" => "Fail", "message" => "missing password");
```
Comment (shalvah) [inline app/Http/Controllers/ApiController.php]: Same here. Let's just call it `$response`.

**Review by shalvah — COMMENTED** [review]
This PR also fixes #3 . Nice!

File: app/Http/Controllers/ApiController.php [hunk 6]
```diff
@@ -36,7 +36,7 @@ public function getStudentDetails(Request $request)
                     "message" => "success");
                 $response["student"] = $this->extractDetails($result);
 
-        return json_encode($response);
+        return response()->json($response, 200);
```
Comment (shalvah) [inline app/Http/Controllers/ApiController.php]: You can omit the 200.

**Comment by VictorAlagwu** [comment]
ok, will work on this

**Comment by VictorAlagwu** [comment]
New commits made, check it out

**Comment by shalvah** [comment]
Nice work.

**Pull request merged**